effects/nixops/run-nixops.nix

```diff
@@ -73,9 +73,16 @@ args@{
   # Defaults to pkgs.path
   NIX_PATH ? "nixpkgs=${path}",
 
+  # run this deployment with nixops deploy --dry-run flag.
+  # eval only, don't do the deploy
+  isDryRun ? false,
```

Comment:
I've though about this some more because I wasn't happy with the "inverted" meaning of dry-run, but I think I've found a bigger issue, which is that combinations with some other flags will be nonsensical. It's better to have a single variable for these mutually exclusive options.

```
runNixOps {
  # ...
  action = if src.ref == "refs/heads/master" then "activate" else "plan";
}
```

Valid values would then be "evaluate", "plan", "build", "create", "copy", "dry-activate", "test", "boot", "activate".

Perhaps "plan" is more useful than "evaluate", which would be like `--dry-run`.

Reply:
Ah I see, so for example:
 
+ `evaluate` would map to an `--evaluate-only` flag, 
+ `plan` to a `--plan-only`
+ `build` to a `--build-only`
+ `create` to a `--create-only`
+ `copy` to a `--copy-only`
+ `dry-activiate` is the same

I don't know where `test`, `boot` and `activiate` should map to. 

It'd be nice becuase as a 
Since it's a single string we can't configure it wrongly.

Reply:
> I don't know where `test`, `boot` and `activiate` should map to.

`activate` is the default behavior.

`test` and `boot` map to

```
  --test                build and activate the new configuration; do not enable it in the bootloader. Rebooting the system will roll back automatically.
  --boot                build the new configuration and enable it in the bootloader; do not activate it. Upon reboot, the system will use the new configuration.
```